### PR TITLE
Update orchestrator build for Python 3.8

### PR DESCRIPTION
**Pull request by paulmelnikow**

### CODE REVIEW

File: werkit/aws_lambda/build.py [hunk 1]
```diff
@@ -45,10 +45,19 @@ def create_venv_with_dependencies(
 
 
 def find_site_packages_dir(venv_dir):
-    result = os.path.join(venv_dir, "lib64", "python3.7", "site-packages")
-    if not os.path.exists(result):
-        result = os.path.join(venv_dir, "lib", "python3.7", "site-packages")
-    return result
+    try:
+        return next(
```
Comment (jbeard4) [inline werkit/aws_lambda/build.py:49]: What is this function `next`, and where is it defined?

Comment (jbeard4) [reply werkit/aws_lambda/build.py:49]: Oh, I see, this is a built-in function, and this is picking out the first path that exists.

**Review by jbeard4 — CHANGES_REQUESTED** [review]
Oh, I see, this is a built-in function, and this is picking out the first path that exists on the filesystem. This was a bit difficult to read, so maybe add a short comment here.

File: werkit/aws_lambda/build.py (hunk 1 above)
Comment (paulmelnikow) [reply werkit/aws_lambda/build.py]: Replaced with a simpler for loop in f78d154.

**Review by jbeard4 — APPROVED** [review]

**Pull request merged**